packages/cfd/src/features/Components/copy-text-icon.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { Clipboard } from '@deriv/components';
+import { localize } from '@deriv/translations';
+import { isMobile } from '@deriv/shared';
+
+type TCopyTextIcon = {
+    text: string | undefined;
```

Comment:
```suggestion
    text: React.ReactNode
```
Already has the both types in it